web/package.json

```diff
@@ -54,11 +54,11 @@
     "@rjsf/antd": "2.2.0",
     "@rjsf/core": "2.2.0",
     "@uiw/react-codemirror": "^3.0.1",
+    "ajv": "^7.0.0-rc.2",
     "antd": "^4.4.0",
     "classnames": "^2.2.6",
     "dayjs": "1.8.28",
     "js-beautify": "^1.13.0",
-    "json-schema": "0.2.5",
```

Comment:
We used an incorrect JSON schema validator 🤔

Reply:
It looks like we used the wrong JSON Schema library.

Reply:
YES